sparkmagic/sparkmagic/tests/test_remotesparkmagics.py

```diff
@@ -269,6 +269,99 @@ def test_run_cell_command_exception():
 
 
 @with_setup(_setup, _teardown)
+def test_run_spark_command_parses():
+    run_cell_method = MagicMock()
+    run_cell_method.return_value = (True, "")
+    spark_controller.run_command = run_cell_method
+
+    command = "-s"
+    name = "sessions_name"
+    context = "-c"
+    context_name = "spark"
+    meth = "-m"
+    method_name = "sample"
+    line = " ".join([command, name, context, context_name, meth, method_name])
+    cell = "cell code"
+
+    result = magic.spark(line, cell)
+
+    run_cell_method.assert_called_once_with(Command(cell), name)
```

Comment:
I'm saying that we don't need to check that the run_cell_method is called and only that the sparkmagicsbase `execute_spark` method is being called with the right parameters. This unit test is here only to check that parsing is correct.